doc/releases/release-notes-3.0.rst

```diff
@@ -287,6 +287,21 @@ USB
 Build and Infrastructure
 ************************
 
+* Build system
```

Comment:
Nothing wrong on this line, but commit message still mentions: `message_if_no_dt_alias()`.

> Mention the dt_alias(), target_sources_if_dt_alias(), and
message_if_no_dt_alias() helpers.